clf_rf.py

```diff
@@ -15,7 +16,15 @@
 plt.ion()
 
 # Hyperopt for hyperparameter optimization did not work well for this dataset.
-def baseline_model(train_data_f, test_data_file, output_dir, compare_models=True, cross_val=False, save=False):
+def baseline_model(
+    train_data_f: str,
+    test_data_file: str,
+    output_dir: str,
+    compare_models: bool = True,
+    cross_val: Literal[None, "simple", "extended"] = None,
```

Comment:
nitpick: 

A more typical way to express this in my experience would be

```suggestion
    cross_val: Optional[Literal["simple", "extended"]] = None,
```